web_src/less/_base.less

```diff
@@ -237,6 +239,12 @@ a:hover,
   background-color: var(--color-secondary-alpha-40);
 }
 
+.ui.ui.ui.ui.table tr.grey:not(.marked),
```

Comment:
this just looks wrong :laughing: , but it works

Reply:
It's copied right from fomantic's stylesheet and we need to match them exactly to properly override. Blame them for using selectors like that 😉.